lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
                     event.preventDefault();
-                    alert("You clicked the button!");
-                    //placeholder; will route to delete team page
+                    var self = this;
+                    var msg = new PromptView.Warning({
```

Comment:
I recommend using ``ViewUtils.confirmThenRunOperation`` instead of constructing the modal yourself.